mne/viz/evoked.py

```diff
@@ -253,6 +277,10 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
                         locs3d = np.array([ch['loc'][:3] for ch in chs])
                         x, y, z = locs3d.T
                         colors = _rgb(x, y, z)
+                        if legend:
+                            pos = find_layout(evoked.info, ch_type=t).pos[:,
+                                                                          :2]
```

Comment:
Very minor, but wouldn't this look better with the line break between `evoked.info` and `ch_type=t` instead?